src/WebJobs.Extensions.Http/Utility.cs

```diff
@@ -26,7 +26,7 @@ internal static void ApplyBindingData(object value, Dictionary<string, object> b
                     // parse the object adding top level properties
                     JObject parsed = JObject.Parse(json);
                     var additionalBindingData = parsed.Children<JProperty>()
-                        .Where(p => p.Value != null && (p.Value.Type != JTokenType.Array))
+                        .Where(p => p.Value != null)
```

Comment:
Note that this same method also exists as a copy in the Functions host.  If we make changes, we need to keep in sync. This code is pretty explicit that the intent is NOT to add arrrays to binding data, and it's been that way since the beginning I believe (check history).

Reply:
Do you remember why we do not want to add collections to binding data? We already add objects. 
Commit:
https://github.com/Azure/azure-functions-host/commit/4d112b27512a507028ded133bc9fde5a4cf259d2#diff-a63b998bf2cabc5184041834fd8678e1